index.html

```diff
@@ -3,39 +3,43 @@
  <head>
    <title>The Gallery of Horns</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <link rel="stylesheet" href="layouts.css">
+   
+   <script id="creature-template" type="text/x-handlebars-template">
+    <section class="card" data-horns="{{horns}}" data-keyword="{{keyword}}">
+      <h2>{{title}}</h2>
+      <img
+        src="{{image_url}}"
+        alt="{{title}}">
+      <p>{{description}}</p>
+    </section>
+  </script>
+
+  <script id="creature-option-template" type="text/x-handlebars-template">
+    <option value="{{keyword}}">{{keyword}}</option>
+  </script>
+

```

Comment:
You should also link to your reset.css (before your other css files), so everything starts from a clean reset.